calculator/dom.js

```diff
@@ -0,0 +1,160 @@
+let buttons = document.querySelectorAll('.button'),
+    numbers = document.querySelectorAll('.number'),
+    operations = document.querySelectorAll('.operation'),
+    decimal = document.getElementById('decimal'),
+    cleaner_all = document.getElementById('ac'),
+    cleaner_window = document.getElementById('c'),
+    display = document.getElementById('display'),
+    sign = document.getElementById('sign'),
+    memoryCurrentNumber = 0,
+    memoryNewNumber = false,
+    memoryError = false,
+    memoryPendingOperation = '';
```

Comment:
one variable declaration - one line. do not use comma separation. 
check this old but [very useful article](https://www.uxebu.com/index.html%3Fp=1135.html)